Source/Mosa.Compiler.Framework/Stages/CILDecodingStage.cs

```diff
@@ -287,6 +286,42 @@ private void InsertExceptionStartInstructions()
 			}
 		}
 
+		protected bool IsSourceAndTargetWithinSameTryOrException(InstructionNode node)
```

Comment:
resolve warning:
`'CILDecodingStage.IsSourceAndTargetWithinSameTryOrException(InstructionNode)': new protected member declared in sealed type`